src/help-panel/implementation.tsx

```diff
@@ -31,7 +31,12 @@ export function HelpPanelImplementation({
   const i18n = useInternalI18n('help-panel');
   const containerProps = {
     ...baseProps,
-    className: clsx(baseProps.className, styles['help-panel'], isToolbar && styles['with-toolbar']),
+    className: clsx(
+      baseProps.className,
+      styles['help-panel'],
+      isToolbar && styles['with-toolbar'],
+      loading && styles.content
```

Comment:
the class name seems not clear if it only applies when `loading` is true. maybe `styles.loading` then?

Reply:
Good call out, I see how this is confusing. Technically the class is used for both, but it is only conditional for loading here because the `.content` class is already provided below in the non-loading version on lines [46/51](https://github.com/cloudscape-design/components/pull/2936/files#diff-64a1630d1728d18eb8064186efc30a870f7ee2e799cd5d7894c7c3f33b09d838R51). However, for clarity sake I created a separate class in the new revision.